modules/sdk-core/src/bitgo/wallet/wallets.ts

```diff
@@ -463,6 +463,14 @@ export class Wallets implements IWallets {
       walletParams.rootPrivateKey = params.rootPrivateKey;
     }
 
+    // Custodial onchain wallets do not need m, n, keys, or keySignatures
+    if (params.type === 'custodial' && params.multisigType === 'onchain') {
```

Comment:
Onchain is typically the default to preserve backwards compatibility. 
```suggestion
    if (params.type === 'custodial' && (params.multisigType ?? 'onchain') === 'onchain') {
```

Reply:
i think you dont need the onchain check, if you got this far, is an onchain multisig wallet